circle.yml

```
@@ -2446,6 +2446,13 @@ windows-workflow: &windows-workflow
     - run-app-integration-tests-chrome:
         name: windows-run-app-integration-tests-chrome
         executor: windows
+        context: test-runner:app-tests
```

This context doesn't exist, it's just test-runner:launchpad-tests:

https://app.circleci.com/settings/organization/github/cypress-io/contexts

```suggestion
        context: test-runner:launchpad-tests
```